pkg/resourceapply/apps_test.go

```diff
@@ -493,6 +497,60 @@ func TestApplyStatefulSet(t *testing.T) {
 			expectedErr:     nil,
 			expectedEvents:  []string{"Normal StatefulSetUpdated StatefulSet default/test updated"},
 		},
+		{
+			name: "deletes and creates the StatefulSet when selector differs",
+			existing: []runtime.Object{
+				newSts(),
```

Comment:
i think this should have a selector set to test the matching later on

Reply:
done